src/cli/src/cluster/check.rs

```diff
@@ -26,7 +27,6 @@ use super::*;
 const MIN_KUBE_VERSION: &str = "1.5.0";
```

Comment:
move these constants to top of the crate like other constants

Reply:
`const MIN_KUBE_VERSION` is at the top where we are declaring all the constants. Where do you suggest to move it?

Reply:
https://github.com/infinyon/fluvio/blob/master/src/cluster/src/install.rs

Reply:
This is related to pre-flight check command:  `fluvio cluster check --pre-install` and logic is in cli.

Reply:
I just want to confirm if this okay. Or do we want to move this logic as well to fluvio-cluster?